root/frontend/src/components/CardColumn.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { useQuery, gql } from '@apollo/client';
+import { VariableSizeList as List } from 'react-window';
+import PreviewCard from './PreviewCard';
+import '../styles/CardColumn.scss';
+import DaySummary from './DaySummary';
+import sizeMap from '../constants/CardColumn';
+
+const GET_WORKOUTS = gql`
+  query getWorkouts {
+    getWorkouts(accountId: "62d64c95cc8ff1221cf7dfa2") {
+      title
+      theme
+      exercises {
+        name
+      }
+    }
+  }
+`;
+
+type WorkoutInfoType = {
+  __typename: string;
+  title: string;
+  theme: string;
+  exercises: ExerciseType[];
+};
+
+type ExerciseType = {
+  __typename: string;
+  name: string;
+};
+
+function CardColumn() {
+  const screenHeight: number = window.innerHeight - 130;
```

Comment:
Might be worth moving this `130` into a constants file of some sort (same applies for any constant numbers which may be referenced elsewhere).